node/directives/validateAdminUserAccess.ts

```diff
@@ -44,7 +45,8 @@ export class ValidateAdminUserAccess extends SchemaDirectiveVisitor {
       const { hasAdminToken, hasValidAdminToken } = await validateAdminToken(
         context,
         adminUserAuthToken as string,
-        metricFields
+        metricFields,
+        orgPermission
```

Comment:
Did you test what happens when the value is different from `'buyer_organization_edit' | 'buyer_organization_view'`?